### ISSUE TITLE
Backend seems to use the id provided in the request body

### ISSUE THREAD

**Issue by nilsweb** [IF11I/backend#16]
When I execute a `PUT` on a component type and omit the id, I get the following error:

```
PHP Notice:  Undefined index: id in /backend/src/routes/routes.php on line 679
PHP Warning:  json_decode() expects parameter 1 to be string, array given in /backend/src/routes/routes.php on line 688
PHP Warning:  Invalid argument supplied for foreach() in /backend/src/routes/routes.php on line 690
```

Please check and correct if so.

**Comment by weinzierl-florian** [comment]
The first line you wrote should be fixed in #17
The other lines you've mentioned in your stack trace are the same issues as in #15

**Comment by nilsweb** [comment]
Closed by #17
